extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -105,10 +117,18 @@ export class AmpAdXOriginIframeHandler {
         this.iframe, 'send-embed-state', true,
         () => this.sendEmbedInfo_(this.baseInstance_.isInViewport()));
 
+    // To provide position to inabox.
+    this.inaboxPositionApi_ = new SubscriptionApi(
```

Comment:
we might consider re-organize the code here (can be a separate PR later). the method is too long